app/scripts/translations.js

```diff
@@ -14,7 +14,7 @@ angular.module('gettext').run(['gettextCatalog', function (gettextCatalog) {
     gettextCatalog.setStrings('nl', {});
```

Comment:
There are few translations with angular binding such as`"{{ errorMessage }}":"{{ Feilmelding }}"` . There are few with <b> tags as well. I wanted to double check if they are ok?